tensornetwork/tensornetwork_test.py

```diff
@@ -1366,7 +1367,16 @@ def test_switch_backend():
 def test_svd_consistency(backend):
   if backend == "pytorch":
     pytest.skip("Complex numbers currently not supported in PyTorch")
-  net = tensornetwork.TensorNetwork(backend=backend)
+  if backend == 'numpy':
```

Comment:
This shouldn't be needed if the `TensorNetwork` automatically sets its backend dependent upon the first node added.